include/ap_mmn.h

```diff
@@ -607,14 +607,15 @@
  *                           and CONN_STATE_PROCESSING
  * 20120211.136 (2.4.59-dev) Add wait_io field to struct process_score
  * 20120211.137 (2.4.59-dev) Add AP_MPMQ_CAN_WAITIO
+ * 20120211.138 (2.4.59-dev) Add is_host_matchable to proxy_worker_shared
```

Comment:
not but I think the version in the comment has been wrong for a few of the recent updates

Reply:
Fixed directly in 2.4.x (and rebased here).